java/src/main/java/com/google/appengine/tools/mapreduce/impl/shardedjob/ShardedJobRunner.java

```diff
@@ -90,17 +74,15 @@ public class ShardedJobRunner<T extends IncrementalTask> implements ShardedJobHa
 
   private static final Logger log = Logger.getLogger(ShardedJobRunner.class.getName());
 
-  static final DatastoreService DATASTORE = DatastoreServiceFactory.getDatastoreService();
-
   // NOTE: no StopStrategy set, must be set by the caller prior to build
   public static RetryerBuilder getRetryerBuilder() {
     return RetryerBuilder.newBuilder()
       .withWaitStrategy(WaitStrategies.exponentialWait(30_000, TimeUnit.MILLISECONDS))
       .retryIfExceptionOfType(ApiProxyException.class)
       .retryIfExceptionOfType(ConcurrentModificationException.class)
-      .retryIfExceptionOfType(DatastoreFailureException.class)
-      .retryIfExceptionOfType(CommittedButStillApplyingException.class)
-      .retryIfExceptionOfType(DatastoreTimeoutException.class)
+      //.retryIfExceptionOfType(DatastoreFailureException.class)
+      //.retryIfExceptionOfType(CommittedButStillApplyingException.class)
+     // .retryIfExceptionOfType(DatastoreTimeoutException.class)
```

Comment:
Now internally handled by the new client, right? Then, I'd add this is as comment to explain why they are commented